src/Avalonia.Diagnostics/Diagnostics/Views/ConsoleView.xaml.cs

```diff
@@ -45,7 +45,7 @@ private void InputKeyDown(object sender, KeyEventArgs e)
             switch (e.Key)
             {
                 case Key.Enter:
-                    vm.Execute();
+                    vm.Execute().RunSynchronously();
```

Comment:
This is changing the behavior of the execution and could potentially hang the process.

Reply:
I revert the change.